index.js

```diff
@@ -38,12 +46,23 @@ var show = ()=>{
 };
 
 //Function to download file from dropbox
-var download = ()=>{
-	return dbx.filesDownload({path: '/texe.jpg'})
+var download = (arg)=>{
+	var dbx = new Dropbox({accessToken: arg.token});
+	return dbx.filesDownload({path: arg.download})
 	.then(res=>{
 		fs.writeFileSync(res.name, res.fileBinary, 'binary', err=>{
 			if(err) throw err;
 		});
-		console.log("Alles gut: " + JSON.stringify(res, null, 2))})
-	.catch(err=>{console.error("Alles schlecht: %s", JSON.stringify(err, null, 2))});
-};
\ No newline at end of file
+		console.log("Done downloading: " + JSON.stringify(res, null, 2))})
+	.catch(err=>{console.error("Something went wrong: %s", JSON.stringify(err, null, 2))});
+};
+
+//Function to diable access token
+var disable = (arg)=>{
```

Comment:
Well, I had to disable my token, and feel someone else might want to... in case it gets compromised 👍